config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/bundle/builder/BundleEntityBuilder.java

```diff
@@ -137,21 +149,35 @@ private Element createDeleteBundle(final Document document, List<Entity> entitie
         return bundleDocumentBuilder.build(document, deleteBundleEntities);
     }
 
+    /**
+     * Creates the DELETE environment bundle element.
+     *
+     * @param document Document
+     * @param entities Entities packaged in the deployment bundle
+     * @return Delete bundle Element for the Annotated Bundle
+     */
+    private Element createDeleteEnvBundle(final Document document, List<Entity> entities) {
+        List<Entity> filteredEntities = copyFilteredEntitiesForDeleteBundle(entities, FILTER_ENV_ENTITIES);
+        filteredEntities.forEach(e -> e.setMappingAction(MappingActions.DELETE));
```

Comment:
Please take care of default listen ports (8080 and 8443) which are part of environment bundle

Reply:
Does env bundle contain other defaults like default identity provider?

Reply:
i am not getting listen ports in the delete bundle as entities are taken from deployment bundle

Reply:
you should generate the environment delete bundle on build-environment-bundle task, otherwise without environment bundle, we will get its delete bundle (i.e on build-bundle task), is that OK?